skglm/solvers/multitask_bcd.py

```diff
@@ -146,7 +150,9 @@ def solve(self, X, Y, datafit, penalty, W_init=None, XW_init=None):
                     if self.ws_strategy == "subdiff":
                         opt_ws = penalty.subdiff_distance(W, grad_ws, ws)
                     elif self.ws_strategy == "fixpoint":
-                        opt_ws = dist_fix_point(W, grad_ws, datafit, penalty, ws)
+                        opt_ws = dist_fix_point(
+                            W, grad_ws, lipschitz, datafit, penalty, ws
```

Comment:
dist_fix_point is really dist_fixed_point_cd (it's algorithm dependent, otherwise it should not depend on the stepsize).

it felt more elegant when the user did not have to pass `lipschitz`, but then again now it's more explicit because you don't run into an error when calling `dist_fix_point` on a datafit which has not been initialized and does not have a `lipschitz` attribute, so I think the current design is safer.

What are your thoughts on renaming `dist_fix_point` (later) ?

Reply:
I'm more in favor of tackling that in a separate PR